public/pages/CreateTrigger/containers/CreateTrigger/utils/formikToTrigger.js

```diff
@@ -334,15 +335,17 @@ export function getResultsPath(isCount) {
   return isCount ? HITS_TOTAL_RESULTS_PATH : AGGREGATION_RESULTS_PATH;
 }
 
-export function getCompositeAggFilter({ where }) {
-  const fieldName = _.get(where, 'fieldName', FORMIK_INITIAL_TRIGGER_VALUES.where.fieldName);
+export function getCompositeAggFilter({ filters = [] }) {
   const composite_agg_filter = {};
-  if (fieldName.length > 0) {
-    composite_agg_filter[where.fieldName[0].label] = {
-      [where.operator]: where.fieldValue,
-    };
-    return composite_agg_filter;
-  }
+  filters.forEach((filter) => {
+    const fieldName = _.get(filter, 'fieldName', FORMIK_INITIAL_WHERE_EXPRESSION_VALUES.fieldName);
+    if (fieldName.length > 0) {
+      composite_agg_filter[fieldName[0].label] = {
+        [filter.operator]: filter.fieldValue,
+      };
+    }
+  });
+  if (!_.isEmpty(composite_agg_filter)) return composite_agg_filter;
```

Comment:
Maybe a dumb question, what would be return if it's empty here

Reply:
If `composite_agg_filter` is empty, the function will return `undefined`; which means the trigger condition will be generated without a `composite_agg_filter` attribute.